### PR TITLE
Feature/pseudonyms part 3

### PR DESCRIPTION
**Pull request by Figments**
## Description

Part 3 of a series of changes aimed at fixing all the problems caused by the move to accounts and pseudonyms. These changes are not yet finished, and there will be a part 4, but this PR was getting too big so i thought it was better to split things up a bit

## Changes

- Partially fixes My Stuff content fetching
- Partially fixes Settings page
- Adjusts navigation UX 

## Areas Affected
- [x] Site Navigation
- [ ] Home
- [x] Sign In/Out
- [x] Registration
- [x] Settings

.
- [ ] Profile
- [x] My Stuff (main page)
- [ ] Editor
- [ ] Browse
- [ ] Work Card

.
- [ ] Work Page
- [ ] Blog Page
- [ ] Collections
- [ ] Comments
- [ ] Other Pages

.
- [x] Account Authentication
- [ ] Dashboard
- [ ] Mobile

### CODE REVIEW

File: apps/api/src/app/guards/identity.guard.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+} from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { TokenExpiredError } from 'jsonwebtoken';
+import { Roles, Account } from '@dragonfish/shared/models/accounts';
+import { JwtPayload } from '@dragonfish/shared/models/auth';
+import { AccountsStore, PseudonymsStore } from '@dragonfish/api/database/accounts/stores';
+import { isAllowed } from '@dragonfish/shared/functions';
+
+/**
+ * This is a mixin so we can do stuff like UseGuards(RolesGuard(['Admin','Moderator'])).
```
Comment (SigmasonicX) [inline apps/api/src/app/guards/identity.guard.ts:18]: Didn't update this comment

File: libs/api/database/src/lib/content/stores/blogs.store.ts [hunk 2]
```diff
@@ -75,45 +74,14 @@ export class BlogsStore {
...
      */
-    async changePublishStatus(user: JwtPayload, blogId: string, pubChange: PubChange): Promise<BlogsContentDocument> {
-        if (pubChange.oldStatus === PubStatus.Unpublished && pubChange.newStatus === PubStatus.Published) {
-            await this.usersService.changeBlogCount(user, true);
-        } else if (pubChange.oldStatus === PubStatus.Published && pubChange.newStatus === PubStatus.Unpublished) {
-            await this.usersService.changeBlogCount(user, false);
-        }
-
+    async changePublishStatus(user: string, blogId: string, pubChange: PubChange): Promise<BlogsContentDocument> {
         return this.blogsModel.findOneAndUpdate(
```
Comment (SigmasonicX) [inline libs/api/database/src/lib/content/stores/blogs.store.ts:78]: There's no longer need to call changeBlogCount()?

File: libs/client/my-stuff/src/lib/components/content-preview/content-preview.component.ts [hunk 3]
```diff
@@ -33,9 +33,13 @@ export class ContentPreviewComponent {
         private stuff: MyStuffService,
         public stuffQuery: MyStuffQuery,
         private router: Router,
-        private alerts: AlertsService
+        private alerts: AlertsService,
     ) {}
 
+    ngOnInit() {
+        this.stuffQuery.current$.subscribe((x) => console.log(x));
+    }
```
Comment (SigmasonicX) [inline libs/client/my-stuff/src/lib/components/content-preview/content-preview.component.ts:41]: If this is just logging, remember to remove later

File: apps/api/src/app/guards/identity.guard.ts (hunk 1 above)
Comment (Figments) [reply apps/api/src/app/guards/identity.guard.ts:18]: can take care of that in part 4

File: libs/api/database/src/lib/content/stores/blogs.store.ts (hunk 2 above)
Comment (Figments) [reply libs/api/database/src/lib/content/stores/blogs.store.ts:78]: that function didn't really work as intended even before this change, so we should handle this plus works count in a later PR.

File: libs/client/my-stuff/src/lib/components/content-preview/content-preview.component.ts (hunk 3 above)
Comment (Figments) [reply libs/client/my-stuff/src/lib/components/content-preview/content-preview.component.ts:41]: yup yup!

**Review by SigmasonicX — APPROVED** [review]
Looks good. Several pages are still broken, but I haven't seen any new breaks.

**Pull request merged**